README.md

```diff
@@ -40,6 +40,13 @@ Your application should configure the following properties for your use case:
 
 This package depends on the messaging library, which has its own set of configurable properties documented [here](https://github.com/OHSU-OCTRI/messaging-lib).
 
+Finally, your application should set these Spring Batch properties to prevent the database tables from being initialized and to prevent batch jobs from running on application restart. This is undesirable behavior for notifications, since they typically run daily at a specific time.
+
+```
+spring.batch.jdbc.initialize-schema=never
+spring.batch.job.enabled=false
+```
+
```

Comment:
I spent way too long (hours) trying to get the library to set these by default, but every iteration failed and the job would try to run on startup of COMPASS unless I explicitly added the properties there. I'm sure there's a way, but they don't make it easy.